tests/agent/test_agent.py

```diff
@@ -39,7 +39,53 @@ def set_required_env_vars(monkeypatch, org_id, token):
 
 
 @pytest.fixture
-def gx_agent_config(queue, connection_string, org_id, token) -> GXAgentConfig:
+def set_required_env_vars_missing_token(monkeypatch, org_id):
+    env_vars = {
+        "GX_CLOUD_ORGANIZATION_ID": org_id,
+        "GX_CLOUD_ACCESS_TOKEN": None,
+    }
+    monkeypatch.setattr(os, "environ", env_vars)
+
+
+@pytest.fixture
+def set_required_env_vars_missing_org_id(monkeypatch, token):
+    env_vars = {
+        "GX_CLOUD_ORGANIZATION_ID": None,
+        "GX_CLOUD_ACCESS_TOKEN": token,
+    }
+    monkeypatch.setattr(os, "environ", env_vars)
```

Comment:
You should use `monkeypatch.setenv` for this kind of thing.

https://docs.pytest.org/en/latest/how-to/monkeypatch.html#monkeypatching-environment-variables

Reply:
nice! ty - updated these tests to use `monkeypatch.setenv` instead